sections/dashboard/SpotHistoryTable/SpotHistoryTable.tsx

```diff
@@ -0,0 +1,284 @@
+import { FC, useMemo, ReactElement } from 'react';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import { useRecoilValue } from 'recoil';
+import { Synths } from '@synthetixio/contracts-interface';
+import { CurrencyKey } from 'constants/currency';
+import Connector from 'containers/Connector';
+import values from 'lodash/values';
+import Currency from 'components/Currency';
+import { CellProps } from 'react-table';
+import Table from 'components/Table';
+import { walletAddressState } from 'store/wallet';
+import useSynthetixQueries from '@synthetixio/queries';
+import { SynthTradesExchangeResult } from '../Transactions/TradeHistory/TradeHistory';
+import useSelectedPriceCurrency from 'hooks/useSelectedPriceCurrency';
+import TimeDisplay from '../../futures/Trades/TimeDisplay';
+import { NO_VALUE } from 'constants/placeholder';
+import BlockExplorer from 'containers/BlockExplorer';
+import { ExternalLink } from 'styles/common';
+import LinkIcon from 'assets/svg/app/link.svg';
+import * as _ from 'lodash/fp';
+import { isFiatCurrency } from 'utils/currencies';
+
+const SpotHistoryTable: FC = () => {
+	const { t } = useTranslation();
+	const walletAddress = useRecoilValue(walletAddressState);
+	const { subgraph } = useSynthetixQueries();
+
+	const { blockExplorerInstance } = BlockExplorer.useContainer();
+	const { selectPriceCurrencyRate, selectedPriceCurrency } = useSelectedPriceCurrency();
+	const walletTradesQuery = subgraph.useGetSynthExchanges(
```

Comment:
@Xiaolei-Shawn The `SpotHistoryTable` is supposed to re-render after the user switches the network (as V1). However, the table now needs to be refreshed manually. Could you please re-check if something is missing here?